features.js

```diff
@@ -1,19 +1,17 @@
 (async () => {
   'use strict';
 
-  function partitionArray(arr, condition) {
-    const matched = [];
-    const unmatched = [];
-
-    for (const item of arr) {
-      if (condition(item)) {
-        matched.push(item);
-      } else {
-        unmatched.push(item);
+  function partitionByPhase(features) {
```

Comment:
I can't comment on the status of proposals, but on the technical side of things, perhaps we can use [`Object.groupBy`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy) here. It's a rather new addition, but I think the assumption was that [developers would use up-to-date browsers](https://github.com/WebAssembly/website/pull/283#issuecomment-1234044185) anyway.

Reply:
I'll look into that tomorrow, thanks.

Reply:
Alright, this is now done.